frontend/src/index.jade

```diff
@@ -38,6 +38,11 @@ html
                 v_authorship(
                   v-bind:key="generateKey(tabInfo.tabAuthorship)",
                   v-bind:info="tabInfo.tabAuthorship")
+              #tab-authorship.tab-pane(v-else)
+                .title
+                  span To get started, click on the &nbsp;
```

Comment:
Change as follows?
~To get started ...~ To view the code attributed to a specific author, click the `</>` icon next to that author's name.

Reply:
the empty tab shouldn't be id-ed `#tab-authorship`, maybe call it `#tab-empty` or something.

cos the idea is that, next time we can add new kinds of side display, say a diff view will have the id `#tab-commit`